tests/unit/test_permissions.py

```diff
@@ -19,16 +19,17 @@
 import pytest
 
 import src.packages.commands.rat_command as Commands
+from config import setup
+from src.packages.cli_manager import cli_manager
```

Comment:
[Style] Both of these appear to be unused imports?

Reply:
quite possibly. this Pr is still a draft so imports have yet to be optimized.